utils/prefsUi.js

```diff
@@ -0,0 +1,47 @@
+const { Gtk } = imports.gi;
```

Comment:
Im not sure about the need of this file. All the functions here will only be called by the pref.js file. The utils file should only contains helper used in several parts of the project